source/common/network/address_impl.cc

```diff
@@ -407,7 +407,7 @@ EnvoyInternalInstance::EnvoyInternalInstance(const std::string& address_id,
                                              const SocketInterface* sock_interface)
     : InstanceBase(Type::EnvoyInternal, sockInterfaceOrDefault(sock_interface)),
       internal_address_(address_id) {
-  friendly_name_ = absl::StrCat("envoy://", address_id);
+  friendly_name_ = absl::StrCat("envoy:", address_id);
```

Comment:
Can we sanitize at https://github.com/envoyproxy/envoy/blob/main/source/server/listener_impl.cc#L220-L224 ?

So we can leave the address, pipe address and internal address untouched.

Reply:
Sure, I wiil sanitize there.